post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.adoc

```diff
@@ -29,7 +25,11 @@ When the operand removes the scheduling gate, the pod enters the scheduling cycl
 
 [IMPORTANT]
 ====
-The Technology Preview version of the Multiarch Tuning Operator does not support clusters with restricted network scenarios.
+If you are using node affinity and node selectors in the same pod configuration, note the following operand behaviors:
+
+* If the `nodeSelector` field is already configured with the `kubernetes.io/arch` label for a workload, the operand does not update the `nodeAffinity` field for that workload.
+
+* If the `nodeSelector` field is not configured with the `kubernetes.io/arch` label for a workload, the operand updates the `nodeAffinity` field for that workload. However, in that `nodeAffinity` field, the operand updates only the `nodeSelectorTerms` fields or `matchExpressions` fields that are not configured with the `kubernetes.io/arch` label.
```

Comment:
```suggestion
* If the `nodeSelector` field is not configured with the `kubernetes.io/arch` label for a workload, the operand updates the `nodeAffinity` field for that workload. However, in that `nodeAffinity` field, the operand updates only the node selector terms that are not configured with the `kubernetes.io/arch` label.
```

`nodeSelectorTerms` is not a visible field to the users: it's a "generic name" that can represent a list of matchFields, matchExpressions, or both. It's a defined API but since it's not visible to the users, should we avoid writing it in camelCase?

Reply:
@aleskandro 

I noticed that `nodeSelectorTerms` is specified similar to other fields in the pod specification.

![Screenshot from 2024-08-27 23-28-14](https://github.com/user-attachments/assets/55a3f7d5-46c8-491b-96c9-ae440cf35794)
https://docs.openshift.com/container-platform/4.16/nodes/scheduling/nodes-scheduler-node-affinity.html

However, I agree with you on using just "nodeSelctorTerms" to refer to all other fields that it can include (such as matchFields, matchExpressions).

Since there are instances in the documentation that use the term `nodeSelectorTerms`, can we keep it this way:

"However, in that `nodeAffinity` field, the operand updates only the `nodeSelectorTerms` that is not configured with the `kubernetes.io/arch` label."

We can also have multiple `nodeSelectorTerms`, in that case we should rather say "`nodeSelectorTerms` fields that are not configured". Can we refer to `nodeSelectorTerms` as a field?